packages/ckeditor5-emoji/src/ui/emojitoneview.ts

```diff
@@ -7,70 +7,69 @@
 	ListItemButtonView,
-
 	View,
 	ViewCollection,
 	DropdownButtonView
 } from 'ckeditor5/src/ui.js';
+import type { Locale } from 'ckeditor5/src/utils.js';
+import type { SkinToneId } from '../emojiconfig.js';
+import type { SkinTone } from '../emojidatabase.js';
 
-export type SkinToneId = 'default' | 'light' | 'medium-light' | 'medium' | 'medium-dark' | 'dark';
-
-type SkinTone = {
-	id: SkinToneId;
-	example: string;
-	tooltip: string;
-};
+import '../../theme/emojitone.css';
 
 export default class EmojiToneView extends View {
-	declare public selectedSkinTone: SkinToneId;
+	/**
+	 * Active skin tone.
+	 *
+	 * @observable
+	 */
+	declare public skinTone: SkinToneId;
 
-	private _skinTones: Array<SkinTone>;
+	/**
+	 * A dropdown element for selecting an active skin tone.
+	 */
+	public readonly mainDropdownButton: DropdownButtonView;
 
-	private _mainDropdownButton: DropdownButtonView;
-	private _dropdownButtons: ViewCollection<ListItemButtonView>;
+	/**
+	 * Option elements to select an active tone.
+	 */
+	public readonly dropdownButtons: ViewCollection<ListItemButtonView>;
+
+	/**
+	 * An array of available skin tones.
+	 */
+	private readonly _skinTones: Array<SkinTone>;
 
 	/**
 	 * @inheritDoc
 	 */
-	constructor( locale: Locale, skinTone: SkinToneId ) {
+	constructor( locale: Locale, { skinTone, skinTones }: { skinTone: SkinToneId; skinTones: Array<SkinTone> } ) {
 		super( locale );
 
 		const t = locale.t;
 
-		this.set( 'selectedSkinTone', skinTone );
-
-		this._skinTones = [
-			{ id: 'default', example: '👋', tooltip: 'Default skin tone' },
-			{ id: 'light', example: '👋🏻', tooltip: 'Light skin tone' },
-			{ id: 'medium-light', example: '👋🏼', tooltip: 'Medium Light skin tone' },
-			{ id: 'medium', example: '👋🏽', tooltip: 'Medium skin tone' },
-			{ id: 'medium-dark', example: '👋🏾', tooltip: 'Medium Dark skin tone' },
-			{ id: 'dark', example: '👋🏿', tooltip: 'Dark skin tone' }
-		];
-
-		this._mainDropdownButton = new DropdownButtonView();
-		const dropdownView = createDropdown( locale, this._mainDropdownButton );
-		this._dropdownButtons = new ViewCollection(
-			this._skinTones.map( ( { id, example, tooltip } ) => this._createButton( locale, id, example, tooltip ) )
+		this.set( 'skinTone', skinTone );
+
+		this._skinTones = skinTones;
+		this.mainDropdownButton = new DropdownButtonView();
+
+		const dropdownView = createDropdown( locale, this.mainDropdownButton );
+
+		this.dropdownButtons = new ViewCollection(
+			this._skinTones.map( ( { id, icon, tooltip } ) => this._createButton( locale, id, icon, tooltip ) )
 		);
 
-		this._mainDropdownButton.withText = true;
-		this._mainDropdownButton.label = this._getSkinTone( this.selectedSkinTone ).example;
-		this._mainDropdownButton.tooltip = 'Select skin tone';
+		this.mainDropdownButton.withText = true;
+		this.mainDropdownButton.label = this._getSkinTone( this.skinTone ).icon;
+		this.mainDropdownButton.tooltip = 'Select skin tone';
 
 		addToolbarToDropdown(
```

Comment:
Semantically a toolbar gives you plenty of options that are independent. A list gives you many options but they are related to a specific domain. 

Is the skin tone a toolbar or a list? I'd say the latter. We're selecting one of possible skin options. There's a `addListToDropdown()` helper that will help you out.

BTW: This also affects accessibility.

Reply:
Fixed in ba94394d82e19e52a7fa5da1fe6b54813eef616c.